_extras/snippets_library/EPCC_Cirrus_pbs/_config_options.yml

```diff
@@ -0,0 +1,19 @@
+# EPCC, The University of Edinburgh: Cirrus + PBS Pro
+host: "Cirrus"
+login_host: "login.cirrus.ac.uk"
+host_location: "EPCC, The University of Edinburgh"
+host_homedir: "/lustre/home/tc001/lola"
+host_prompt: "[yourUsername@cirrus-login0 ~]$"
+sched_name: "PBS Pro"
+sched_submit: "qsub"
+sched_submitopt: "-A tc001 -q R387726"
+sched_stat: "qstat"
+sched_uopt: "-u yourUsername"
+sched_del: "qdel"
```

Comment:
`sched_statu` is undefined.

Reply:
@tkphd Fixed in https://github.com/hpc-carpentry/hpc-intro/pull/101/commits/579a4bdd87c290832744133096f211a6555b24bb